src/app/app.module.ts

```diff
@@ -1,5 +1,6 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule } from '@angular/forms';
```

Comment:
Is this import necessary? because I think it is being imported into the other module